plugins/search/src/index.ts

```diff
@@ -28,20 +28,22 @@ export {
   SearchContextProvider,
   SearchFilter,
   SearchFilterNext,
+  SearchModal,
   SearchPage as Router,
   SearchResultPager,
   SearchType,
   SidebarSearch,
+  SidebarSearchModal,
```

Comment:
I might recommend, rather than exporting the component directly here, that you export this as a component extension in `plugin.ts`.  Something like:

```tsx
export const SidebarSearchModal = searchPlugin.provide(
  createComponentExtension({
    component: {
      lazy: () =>
        import('./components/SidebarSearchModal').then(m => m.SidebarSearchModal),
    },
  }),
);
```

^ Gets us a nice error boundary, lazy loading, and more.